From f724e412289b7e21327e16b66b23cc8ec648e0ba Mon Sep 17 00:00:00 2001
From: hwg <hwg@rock-chips.com>
Date: Sat, 28 Feb 2015 13:59:24 +0800
Subject: [PATCH] support different wifi bt chip auto compatible  Attention:
 this is only for android 5.0, not compatible for android 4.4, meaning can't
 compatible for 4.4 after this commit  modify
 arch/arm/boot/dts/rk3288-tb_8846.dts

---
 arch/arm/boot/dts/rk3288-tb_8846.dts | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-tb_8846.dts b/arch/arm/boot/dts/rk3288-tb_8846.dts
index d013f0694513..6362cd2b16dd 100755
--- a/arch/arm/boot/dts/rk3288-tb_8846.dts
+++ b/arch/arm/boot/dts/rk3288-tb_8846.dts
@@ -13,13 +13,12 @@
     wireless-wlan {
         compatible = "wlan-platdata";
 
-        /* wifi_chip_type - wifi chip define
-         * bcmwifi ==> like ap6xxx, rk90x;
-         * rtkwifi ==> like rtl8188xx, rtl8723xx;
-         * esp8089 ==> esp8089;
-         * other   ==> for other wifi;
-         */
-        wifi_chip_type = "bcmwifi";
+		/* wifi_chip_type - wifi chip define
+                 * ap6210, ap6330, ap6335
+                 * rtl8188eu, rtl8723bs, rtl8723bu
+                 * esp8089
+		*/
+		wifi_chip_type = "ap6335";
 
         sdio_vref = <1800>; //1800mv or 3300mv
 
-- 
2.35.3

